tools/debian_config.json

```diff
@@ -0,0 +1,30 @@
+{
+    "maintainer_name":"Microsoft",
```

Comment:
@Eilon Please review this debian config file. This is mostly copied over from https://github.com/dotnet/cli/blob/3419a87d6fb620a2ea7981cdd6cd43db3b17cb5f/packaging/deb/dotnet-debian_config.json

Nodes especially important are maintainer_*, maintainer_email, package_name, install_root, *_description, homepage, copyright and license. The values in release and control seem to be dummy values though I need to verify them.